web/client/epics/featuregrid.js

```diff
@@ -242,29 +258,23 @@ module.exports = {
                         .take(1)
                     )
             ),
+    featureGridUpdateGeometryFilter: (action$, store) =>
```

Comment:
I think you should put this logic in the `featureGridUpdateFilter` and/or document it. This way the update logic is all together and documented. Splitting it into different epics decrese readability, if you don't notice that the two filters are mutually exclusive.